Notes and Outlines/markdown modules/markdown.md

```diff
@@ -0,0 +1,302 @@
+Markdown is a simple, human-readable markup language. In other words, it's a way to format plaintext in a way that is easy to read and write as plaintext, and which a program like Jekyll can then take and format easily.
+
+A lot of sites use markdown. You may have seen it in blog comments, reddit posts, or GitHub issues. It's popular because it's so easy to use. GitHub Pages and Jekyll use markdown to define the content of websites so you don't have to write HTML and CSS unless you're redesigning the site.
+
+In this lesson, we're going to go over the basics of markdown, as well as some GitHub-specific features. By the end, you'll be able to use markdown not only to produce beautiful Jekyll sites, but also around the web.
+
```

Comment:
This lesson looks awesome. I wish I had this earlier. Our team just rewrote the ["Writing on GitHub"](https://help.github.com/categories/writing-on-github/) docs in case you want to link out to more info. 

Also, I found this Markdown tutorial a while ago and thought it looked cool: http://commonmark.org/help/. I am excited to see how this content will work on Wheehouse.io.